Artem.Arkhytiuk/Lesson2/index.css

```diff
@@ -1,133 +1,241 @@
+
+.main {
+  padding-top: 30px;
 }
 
-body{
-  margin: 0;
+.flexContainer {
+  display: flex;
+  align-items: center;
+  width: 100%;
 }
 
+h1 {
+  margin: 0;
+}
 
 /*##########################Navbar#################################*/
 
-.pageHeader{
-  width: 100%;
+.pageHeader {
   overflow: hidden;
   position: fixed;
   z-index: 1000;
+  background-color: #292b2c;
+  width: 100%;
 }
 
-.pageHeader nav{
-  background-color: #c7c9cc; 
-}
-
-.pageHeader nav a{
+.pageHeader nav a {
   text-decoration: none;
   padding: 10px;
-  display: inline-block;
-  font-size: 17px;
+  font-size: 15px;
   text-transform: uppercase;
-  color: #818384;
+  color: #ffffff;
+  font-family: sans-serif;
 }
 
-header.pageHeader nav a:hover{
+.navigationRight {
+  margin-left: auto;
+}
+
+header.pageHeader nav a:hover {
   background-color: white;
   color: black;
 }
 
-.topFigure{
+.topFigure {
   margin: 0;
-  padding-top: 30px;
   overflow: hidden;
 }
 
 /*##########################Article#################################*/
 
-article{
+article {
   line-height: 1.5;
-  padding-left:15%;
+  padding-left: 15%;
   padding-right: 15%;
 }
 
-article header h1{
+article header h1 {
   text-align: center;
   font-family: 'Mina', sans-serif;
 }
 
-article h2{
+article h2 {
   font-family: 'Mina', sans-serif;
   text-align: center;
 }
 
-article p{
+article p {
   font-family: 'Crimson Text', serif;
   font-size: 20px;
   text-align: justify;
 }
 
-main article img{
+main article img {
   margin: 10px;
   height: 350px;
 }
 
-.leftImg{
+.leftImg {
   float: left;
 }
 
-.rightImg{
+.rightImg {
   float: right;
   position: relative;
   height: 150px;
 }
 
 .monospacedText {
   background-color: #f1f1f1;
-  box-shadow:  0 0 0 1px rgba(0,0,0,.15);
+  box-shadow: 0 0 0 1px rgba(0, 0, 0, .15);
   border-radius: 5px;
   width: 615px;
   margin-left: 10px;
 }
-.monospacedText pre{
+
+.monospacedText pre {
   padding: 4px;
   font-family: 'Roboto Mono', monospace;
 }
 
 /*##########################Footer#################################*/
-footer{
+
+footer {
+  display: flex;
   background-color: #666;
-  float: left;
-  width:100%;
+  width: 100%;
+  justify-content: space-between;
 }
 
-footer section{
-  display: inline-block;
+footer section {
+  display: flex;
+  align-items: center;
 }
 
-.leftFooter a{
+.leftFooter a {
   text-decoration: none;
   color: #fff;
-  font-size: 20px;
+  font-size: 15px;
   padding: 10px;
-  display: inline-block;
+  font-family: sans-serif;
 }
 
-.socialFooter{
-  float: right;
-  vertical-align: middle;
-  padding: 5px 2px 0 0;
+.socialFooter {
+  align-items: center;
 }
 
-.socialFooter a{
-  display: inline-block;
+.socialFooter a {
   padding: 2px;
 }
 
-.socialFooter a img{
-  padding: 0;
+.socialFooter a img {
   height: 35px;
 }
+
+/*##########################About Us#################################*/
+
+.mainSection {
+  padding-top: 33px;
+  background-color: #51546b;
+  color: white;
+  height: 94.8vh;
```

Comment:
I assume it's quite rough hardcoded value which is very unreliable :) 
Maybe you need to use a `calc` here instead? :)